package.json

```diff
@@ -72,11 +73,9 @@
     "eslint-plugin-react": "7.20.6",
     "gulp": "4.0.2",
     "gulp-postcss": "8.0.0",
-    "gulp-prettier": "3.0.0",
```

Comment:
Was this not being used or was it only used by Theo before?

Reply:
It was only used by Theo